disposebag/src/main/kotlin/com/kizitonwose/android/disposebag/DisposeBag.kt

```diff
@@ -44,22 +38,28 @@ class DisposeBag @JvmOverloads constructor(owner: LifecycleOwner,
         composite.dispose()
     }
 
-    override fun add(d: Disposable) = composite.add(d)
+    override fun add(disposable: Disposable) = composite.add(disposable)
 
-    override fun remove(d: Disposable) = composite.remove(d)
+    override fun remove(disposable: Disposable) = composite.remove(disposable)
```

Comment:
This can be merged but will cause issues for people who already specify the parameter name when calling the method, making it breaking change. Since the method takes only one parameter, I think we can retain the old `d` parameter name which is from the `DisposableContainer` interface method, maybe until the next major release.

Reply:
I get your point here, if you think there are people already using the library then this will be a valid concern. But then the update will be in different update and will only break for those that upgrade to new version of the library.